blockonomics-woocommerce.php

```diff
@@ -242,123 +267,176 @@ function show_basic() {
-                    </tr>
-                    <tr valign="top">
-                        <th scope="row">Destination BTC wallet for payments</th>
-                        <td>
-                            <?php
-                            $total_received = get_option('blockonomics_temp_withdraw_amount') / 1.0e8;
-                            $api_key = get_option("blockonomics_api_key");
-                            $temp_api_key = get_option("blockonomics_temp_api_key");
-                            if ($temp_api_key && !$api_key && !($total_received > 0)): ?>
-
-                            <p><b>Blockonomics Wallet</b> (Balance: 0 BTC)</p>
-                            <p>We are using a temporary wallet on Blockonomics to receive your payments.</p>
-                            <p>To receive payments directly to your wallet (recommended) -> Follow Wizard by clicking on <i>Get Started for Free</i> on <a href="https://www.blockonomics.co/merchants" target="_blank">Merchants</a> and enter the APIKey below [<a href="https://blog.blockonomics.co/how-to-accept-bitcoin-payments-on-woocommerce-using-blockonomics-f18661819a62">Blog Instructions</a>]</p>
-
-                            <?php elseif ($temp_api_key && $total_received > 0): ?>
-
-                            <p><b>Blockonomics Wallet</b> (Balance: <?php echo "$total_received"; ?> BTC)</p>
-                            <?php if (!$api_key): ?>
-                            <p> To withdraw, follow wizard by clicking on <i>Get Started for Free</i> on <a href="https://www.blockonomics.co/merchants" target="_blank">Merchants</a>, then enter the APIKey below [<a href="https://blog.blockonomics.co/how-to-accept-bitcoin-payments-on-woocommerce-using-blockonomics-f18661819a62">Blog Instructions</a>]
-                            </p>
-                            <?php else: ?>
-                            <p> To withdraw, Click on <b>Test Setup</b></p>
+                <input type="hidden" name="blockonomics_redirect" id="blockonomics_redirect" value="false">
+                <?php wp_nonce_field('update-options');
+                switch ( $active_tab ){
+                case 'settings' :?>
+                <div class="bnomics-width">
+                    <h4><?php echo __('API Key', 'blockonomics-bitcoin-payments')?></h4>
+                    <input class="bnomics-options-input" onchange="value_changed()" size="130" type="text" id="blockonomics_api_key" name="blockonomics_api_key" value="<?php echo get_option('blockonomics_api_key'); ?>" />
+                    <p class='bnomics-options-no-margin'><?php echo __('To get your API Key, click <b> Get Started for Free </b> on', 'blockonomics-bitcoin-payments')?>
+                        <a href="https://blockonomics.co.merchants"><?php echo __('https://blockonomics.co.merchants', 'blockonomics-bitcoin-payments')?></a>
```

Comment:
The links to Blockonomics should open in a new browser tab. Add `target="_blank"` to the Get started links.

Reply:
Done.